srcs/juloo.keyboard2/Keyboard2.java

```diff
@@ -124,6 +125,46 @@ public void onCreate()
     _foldStateTracker.setChangedCallback(() -> { refresh_config(); });
   }
 
+  @Override
+  public boolean onKeyDown(int keyCode, KeyEvent event) {
+    int inputClass = _lastInputType & InputType.TYPE_MASK_CLASS;
+    InputConnection conn = getCurrentInputConnection();
+    if (conn == null || !isInputViewShown() || inputClass != InputType.TYPE_CLASS_TEXT) {
```

Comment:
Do you think the `_lastInputType` thing could be avoided if this code was moved to `Keyboard2View` ?